docs/source/configuration/telemetry/instrumentation/selectors.mdx

```diff
@@ -75,15 +75,15 @@ The subgraph service executes multiple times during query execution, with each e
 |-----------------------------|-------------|------------------|--------------------------------------------------------------------------------|
 | `subgraph_operation_name`   | Yes         | `string`\|`hash` | The operation name from the subgraph query                                     |
 | `subgraph_operation_kind`   | No          | `string`         | The operation kind from the subgraph query                                     |
-| `subgraph_query`            | Yes         | `string`         | The graphql query to the subgraph                                              |
+| `subgraph_query`            | Yes         | `string`         | The GraphQL query to the subgraph                                              |
 | `subgraph_name`             | No          | `true`\|`false`  | The subgraph name                                                              |
 | `subgraph_query_variable`   | Yes         |                  | The name of a subgraph query variable                                          |
-| `subgraph_response_data`    | Yes         |                  | Json Path into the subgraph response body data (it might impact performance)   |
-| `subgraph_response_errors`  | Yes         |                  | Json Path into the subgraph response body errors (it might impact performance) |
+| `subgraph_response_data`    | Yes         |                  | JSON path into the subgraph response body data (it might impact performance)   |
```

Comment:
```suggestion
| `subgraph_response_data`    | Yes         |                  | JSON Path into the subgraph response body data (it might impact performance)   |
```